connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/webhook/WebhookConnectorRegistry.java

```diff
@@ -16,20 +16,29 @@
  */
 package io.camunda.connector.runtime.inbound.webhook;
 
+import io.camunda.connector.api.inbound.Activity;
+import io.camunda.connector.api.inbound.Severity;
 import io.camunda.connector.runtime.inbound.lifecycle.ActiveInboundConnector;
 import io.camunda.connector.runtime.inbound.webhook.model.CommonWebhookProperties;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Optional;
+import java.util.regex.Pattern;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class WebhookConnectorRegistry {
 
+  private static final String WEBHOOK_CONNECTOR_REGISTRY = "WebhookConnectorRegistry";
+  public static final String DEPRECATED_WEBHOOK_MESSAGE_PREFIX = "Deprecated webhook path: ";
```

Comment:
Thanks for this suggestion!

Who would be the audience of this log message? 

Currently, this PR's code only logs using `InboundConnectorRestController.log()` which is accessible via the `/inbound` REST API - perhaps we should log elsewhere or in an additional place.

Even if we keep the coding logic the same, I agree that adding more context would be helpful :)

Reply:
These logs will be visible in the web modeler, under "Last activities" 🙂 

<img width="390" alt="Screenshot 2024-01-24 at 16 06 42" src="https://github.com/camunda/connectors/assets/38818382/1d7500de-6b29-4a7c-84b4-725d41c69763">

Reply:
Thanks for this context!

Since that Webhook panel also contains more technical debugging info like the HTTP method, full URL's, and unformatted dates, I'm assuming the panel is intended for a more technical persona.

If my assumption is right, then I'm ok with putting a regex pattern in the log message.

It also doesn't seem like we need to log anywhere else.

Here's what the new log entry looks like from the inbound logs endpoint:
```
[[{"severity":"WARNING","tag":"WebhookConnectorRegistry","timestamp":1706114818.142828000,"message":"Deprecated webhook path: €. This may lead to unexpected behavior. Consider adjusting the path to match the pattern: ^[a-zA-Z0-9]([a-zA-Z0-9_-]*[a-zA-Z0-9])?$"}]]
```